app/v1/support/DesResponseMappingSupport.scala

```diff
@@ -20,14 +20,14 @@ import utils.Logging
 import v1.controllers.EndpointLogContext
 import v1.models.errors._
 import v1.models.outcomes.ResponseWrapper
-import v1.models.response.listPayments.ListPaymentsResponse
+import v1.models.response.listPayments.{ListPaymentsResponse, Payment}
 
 trait DesResponseMappingSupport {
   self: Logging =>
 
   final def validateListPaymentsSuccessResponse[T](desResponseWrapper: ResponseWrapper[T]): Either[ErrorWrapper, ResponseWrapper[T]] = {
     desResponseWrapper.responseData match {
-      case listPaymentsResponse: ListPaymentsResponse if listPaymentsResponse.payments.isEmpty =>
+      case listPaymentsResponse: ListPaymentsResponse[Payment] if listPaymentsResponse.payments.isEmpty =>
```

Comment:
`non-variable type argument v1.models.response.listPayments.Payment in type pattern v1.models.response.listPayments.ListPaymentsResponse[v1.models.response.listPayments.Payment] is unchecked since it is eliminated by erasure
case listPaymentsResponse: ListPaymentsResponse[Payment] if listPaymentsResponse.payments.isEmpty =>
` warning when running tests.